features/desktop/donation_form_values_get_passed_correctly_to_the_donation_page.js

```diff
@@ -0,0 +1,27 @@
+import {amounts, intervals, paymentTypes, upgradeOptions} from '../../config/banners.js';
+import { amount, interval, paymentType } from '../../config/forms.js';
+import DonationForm from "../../pages/DonationForm.js";
+
+export default class DonationFormValuesGetPassedCorrectlyToTheDonationPage {
```

Comment:
I'm not sure why these are classes, an anonymous object could be returned here. If keeping a name is important then you could do something like:

`export default const donationFormValuesGetPassedCorrectlyToTheDonationPage {};`

But I think the description covers this.